### PR TITLE
[Enhancement] (nereids)implement showTabletBelongCommand in nereids

### LINKED ISSUE #42758: [Enhancement] (nereids)implement showTabletBelongCommand in nereids
showTabletBelong command is supported in legacy planner but not in nereids(new doris fe planner). you need implement the following showTabletBelong command in nereids:
unsupportedshowTabletBelongStatement
``` SHOW TABLETS BELONG
        tabletIds+=INTEGER_VALUE (COMMA tabletIds+=INTEGER_VALUE)*                  ```

### PR DESCRIPTION
**Pull request by Vallishp**
### What problem does this PR solve?

Issue Number: close #42758 

Related PR: #xxx

Problem Summary:

### Release note

None

### Check List (For Author)

- Test 
    - [ ] Regression test //updated
    - [ ] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [ ] Other reason 

- Behavior changed:
    - [ ] No.
    - [ ] Yes. 

- Does this need documentation?
    - [ ] No.
    - [ ] Yes. 

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

**Comment by Vallishp** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/ShowTabletsBelongCommand.java [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+            }
+            line.add(new Long(totalBucketNum).toString());
+            line.add(new Long(olapTable.getReplicaCount()).toString());
+            line.add(tableToTabletIdsMap.get(tableId).toString());
+
+            rows.add(line);
+        }
+        return new ShowResultSet(getMetaData(), rows);
+    }
+
```
Comment (starocean999) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/ShowTabletsBelongCommand.java:143]: need override toRedirectStatus method like old planner
```
    @Override
    public RedirectStatus getRedirectStatus() {
        return RedirectStatus.FORWARD_NO_SYNC;
    } 
```

**Comment by Vallishp** [comment]
run buildall

**Comment by Vallishp** [comment]
run p0

**Review by starocean999 — APPROVED** [review]

**Review by LiBinfeng-01 — APPROVED** [review]

**Pull request merged**